### PR TITLE
fix(client): reintroduce module commands

### PR DESCRIPTION
**Pull request by elsirion**

### CODE REVIEW

**Review by shaurya947 — APPROVED** [review]

File: devimint/src/main.rs [hunk 1]
```diff
@@ -388,6 +388,19 @@ async fn cli_tests(dev_fed: DevFed) -> Result<()> {
         .unwrap();
     assert_eq!(client_ng_reissue_amt, reissue_amount);
 
+    // Ensure that client can reissue via module commands
+    info!("Testing reissuing e-cash via module commands");
+    let reissue_notes = cmd!(fed, "spend", reissue_amount).out_json().await?["notes"]
+        .as_str()
+        .map(|s| s.to_owned())
+        .unwrap();
+    let client_ng_reissue_amt = cmd!(fed, "module", "--module", "mint", "reissue", reissue_notes)
```
Comment (dpc) [inline devimint/src/main.rs:397]: If we are picking module by "kind", shouldn't it be `--kind mint` (vs `--instance 1`)?

Comment (dpc) [reply devimint/src/main.rs:397]: Oh... it does auto-detection... I might have even implemented it... :rofl: . Nvmd me.

**Review by dpc — APPROVED** [review]

File: devimint/src/main.rs (hunk 1 above)
Comment (elsirion) [reply devimint/src/main.rs:397]: I was wondering the same but too lazy to change the behavior since the existing one was good enough.

**Comment by dpc** [comment]
Seems backward compatible.

**Pull request merged**